docs/document/website/siteConfig.js

```diff
@@ -52,14 +52,15 @@ const siteConfig = {
   blogSidebarCount: "ALL",
   blogSidebarTitle: { default: "Recent posts", all: "All posts" },
   scripts: [
+    "/js/gmtHeader.js",
```

Comment:
The name of the service is Google Tag Manager, so the file name will be gtmHeader.js